operations/security/product-security/product-vulnerability-process.md

```diff
@@ -0,0 +1,209 @@
+
+## Process Definitions
+
+### Responsible Parties
+- Product Security Team: Process Ownership; Vulnerability Owner
+- Development Team: Vulnerability Remediation
+- Release Manager: Release Planning
+- Build Team: Build Process
+
+### Vulnerability Sources
+Vulnerabilities are reported from a variety of sources that are explained below.
+
+#### Internal Finding
+Vulnerability has been discovered by the internal security, development, or QA team. Additional possibility is finding 
+through internal security tooling such as OWASP DepCheck.
+
+#### Responsible Disclosure
+Vulnerability has been reported through our responsible disclosure program.
+
+#### Bug Bounty Program
+Vulnerability has been reported through our Bug Bounty program running on HackerOne.
+
+#### Customer Report
+Vulnerability has been reported from customers as part of an internal audit or on-going operations.
+
+#### Public Leak
+Vulnerability has not been officially reported, but released to the public without any notice. It is possible that it 
+can contain reproduction steps or not.
+
+### Vulnerability Severity
+Vulnerabilities must be scored according to the Common Vulnerability Scoring System Version 3.1 (CVSS 3.1). The CVSS 
+is an industry-wide accepted standard to determine the impact of vulnerabilities in software products and takes impacts 
+such as attack vector, complexity, user interaction, scope, and various other components into consideration.
+
+FIRST provides a CVSS calculator at the following URL: https://www.first.org/cvss/calculator/3.1
+
+The CVSS 3.1 provides a textural representation of the severity score in Section 5 of the Specification Document. The 
+mapping is included below:
+
+<table>
+  <thead>
+    <tr>
+      <th style="text-align:left">Rating</th>
+      <th style="text-align:left">CVSS Score</th>
+    </tr>
+  </thead>
+  <tbody>
+        <tr>
+            <td>None</td>
+            <td>0.0</td>
+        </tr>
+        <tr>
+            <td>Low</td>
+            <td>0.1 - 3.9</td>
+        </tr>
+        <tr>
+            <td>Medium</td>
+            <td>4.0 - 6.9</td>
+        </tr>
+        <tr>
+            <td>High</td>
+            <td>7.0 - 8.9</td>
+        </tr>
+        <tr>
+            <td>Critical</td>
+            <td>9.0 - 10.0</td>
+        </tr>
+  </tbody>
+</table>
+
+### Remediation SLAs
+
+(INTERNAL - TBD to Publish)
+
+### Issue Types
+Vulnerabilities must be assigned one of the following vulnerability types:
+- Authorization
+- Reducing Attack Surface
+- XSS
+- Input Validation
+- Phishing
+- Injection
+- Information Disclosure
+- DOS
+- Authentication
+- CSRF
+- Misconfiguration
+- 3rd Party Library
+- Remote Code Execution
+
+If no fitting category can be found, the security engineer must request the inclusion of a new type from the product 
+security team lead.
+
+### Backport Policy
+
+#### Vulnerabilities with Low Score to Medium Score:
+- Upcoming Release
+- Extended Support Release
```

Comment:
Let me know if this is a new policy change. So far the policy has been the following (or what I was instructed to do when I took over release management):

 - Medium level vulnerabilities have the same backport policy as High/Critical level ones.
 - Low level vulnerabilities don't need to be backported.